server/src/api.py

```diff
@@ -176,6 +176,20 @@ def run_user_script(script):
     return res
 
 
+@jsonrpc.method('App.list_suites')
```

Comment:
Tutaj endpoincik który powinien zwracać listę suit pogrupowaną wg tych katalogów które sobie wpisze user w `.prepcase.json`. Ścieżki suit muszą być absolute, bo to potem będzie parametr do endpointa `get_suite` zwracającego szczegóły suity